skimage/registration/_optical_flow.py

```diff
@@ -67,6 +67,7 @@ def _tvl1(
         sparse=True,
     )
 
+    # dt corresponds to tau in [1]_, i.e. the time step
```

Comment:
Hi, I am afraid I got confused because Part 1 of #7313 refers to paper 1 in the docstring (for instance in the definition of tightness). You are right, in the comment related to [3] is the correct reference considering the table reproduced. I am happy to change it.